### PR TITLE
creating new branch

### PR DESCRIPTION
**Pull request by Priyanshu0713**

### CODE REVIEW

**Comment by Priyanshu0713** [comment]
ggg

File: components/hashtaghome2.js [hunk 1]
```diff
@@ -2,53 +2,43 @@ import Head from "next/head";
 import Link from "next/link";
 import Image from "next/image";
 import hashtag from "../assets/hashtag.jpg";
-import "../styles/404.css";
```
Comment (Rishuhashtag) [inline components/hashtaghome2.js:5]: Please remove  unused path

File: components/hashtaghome2.js [hunk 2]
```diff
@@ -2,53 +2,43 @@ import Head from "next/head";
...
 import mf_avatar from "../assets/mf_avatar.jpg";
 import computer from "../assets/computer.JPG";
 import one from "../assets/one.jpg";
 import two from "../assets/two.jpg";
 import three from "../assets/three.jpg";
-import MessageTwoTone from "@mui/icons-material/MessageTwoTone";
 import { makeStyles } from "@material-ui/core/styles";
-import Header from "./header";
```
Comment (Rishuhashtag) [inline components/hashtaghome2.js:13]: Please remove unwanted line 11,  13-17

File: components/hashtaghome2.js [hunk 3]
```diff
@@ -2,53 +2,43 @@ import Head from "next/head";
...
+import Divider from '@mui/material/Divider';
+import ListItem from '@mui/material/ListItem';
+import ListItemButton from '@mui/material/ListItemButton';
+import ListItemIcon from '@mui/material/ListItemIcon';
+import ListItemText from '@mui/material/ListItemText';
+import InboxIcon from '@mui/icons-material/MoveToInbox';
+import MailIcon from '@mui/icons-material/Mail';
+import MenuOpenIcon from '@mui/icons-material/MenuOpen';
 
 const useStyles = makeStyles({
```
Comment (Rishuhashtag) [inline components/hashtaghome2.js:28]: Remove not used line 28 , 32, 39

File: components/hashtaghome2.js [hunk 4]
```diff
@@ -166,105 +131,52 @@ const useStyles = makeStyles({
       },
     },
   },
```
Comment (Rishuhashtag) [inline components/hashtaghome2.js:133]: please keep closing curly bracket in same column line

File: components/hashtaghome2.js [hunk 5]
```diff
@@ -297,47 +205,36 @@ blue_bar_content:{
...
         
         maxWidth: '500px',
         minWidth: '350px',
         height: '830px',
-        // background: 'blue',
         margin: '2px',
-        // color: 'white',
         width: '100%',
         '@media screen and (max-width: 1070px)': {
             minWidth: '280px',
```
Comment (Rishuhashtag) [inline components/hashtaghome2.js:240]: please align in same column line , from line 343 to 294

File: components/hashtaghome2.js [hunk 6]
```diff
@@ -442,55 +334,34 @@ mentor_div2:{
...
-
  // >>>>>>>>>>>>>>>  Interested  >>>>>>>>>>>>>>>>>>>>>>>>>>
 
  interest_hash:{
-  // position: "absolute",
-  // bottom: "-270%",
-  // left: "40%",
+  backgroundColor:"white",
   textAlign: "center",
   margin:"150px 10px",
```
Comment (Rishuhashtag) [inline components/hashtaghome2.js:343]: please remove this line including line 353, 356, 358,360, 375,377,

File: components/hashtaghome2.js [hunk 7]
```diff
@@ -502,54 +373,66 @@ start: {
...
+    <div
+      sx={{
+        width: drawerWidth,
+        flexShrink: 0,
+        [`& .MuiDrawer-paper`]: { width: drawerWidth, boxSizing: 'border-box' },
+      }}
+      role="presentation"
+      onClick={toggleDrawer(anchor, false)}
+      onKeyDown={toggleDrawer(anchor, false)}
+    >
```
Comment (Rishuhashtag) [inline components/hashtaghome2.js:405]: please shift this to in line 404

**Review by im-nikhilkatiyar — APPROVED** [review]

**Pull request merged**